src/Entity/JobOffer.php

```diff
@@ -19,6 +19,9 @@ class JobOffer
     const STATUS_PENDING_REVIEW = 'PENDING_REVIEW';
     const STATUS_APPROVED = 'APPROVED';
 
+    const HIRING_TYPE_CLT = 'CLT';
```

Comment:
It would be nice if we use Enum instead of constants. Will approve BTW

Reply:
Sorry I did similar the `status` field

Reply:
Ok. Will put it to be done in another moment